composer.json

```diff
@@ -26,7 +26,6 @@
 		"nette/caching": "~2.3@dev",
 		"nette/component-model": "~2.2@dev",
 		"nette/database": "~2.3@dev",
-		"nette/deprecated": "~2.3@dev",
```

Comment:
Removed because of `Fatal error: Cannot use Nette\Utils\ObjectMixin as ObjectMixin because the name is already in use`.